pds4indextools/template_pds.xml

```diff
@@ -0,0 +1,178 @@
+$END_IF
+$IF(Product_Metadata_Supplemental)   
+<Product_Metadata_Supplemental xmlns="http://pds.nasa.gov/pds4/pds/v1"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1L00.xsd">
+$END_IF
+    <Identification_Area>
+        <logical_identifier>$logical_identifier$</logical_identifier>                          
+        <version_id>$version_id$</version_id> <!--user input, optional-->
+        <title>$title$</title>           <!--user input, optional-->
+        <information_model_version>1.21.0.0</information_model_version>
+        <product_class>Product_Ancillary</product_class>
+   $IF(Citation_Information)
+       <Citation_Information>          <!--user input, optional-->
+            <author_list>$Citation_Information['author_list']$</author_list> <!--user input, optional-->
+            <editor_list>$Citation_Information['editor_list']$</editor_list> <!--user input, optional-->
+            <publication_year>$Citation_Information['publication_year']$</publication_year> <!--user input, optional-->
+            <doi>$Citation_Information['doi']$</doi> <!--user input, optional-->
+        $FOR(Citation_Information['keyword'])
+            <keyword>$VALUE$</keyword> <!--user input, optional-->
+        $END_FOR
+            <description>$Citation_Information['description']$</description> <!--user input, optional-->
+        $IF(Citation_Information.get('Funding_Acknowledgement']))
+            <Funding_Acknowledgement><!--user input, optional-->
+                <funding_source>$Funding_Acknowledgement['funding_source']$</funding_source> <!--user input, optional-->
+                <funding_year>$Funding_Acknowledgement['funding_year']$</funding_year> <!--user input, optional-->
+                <funding_award>$Funding_Acknowledgement['funding_year']$</funding_award> <!--user input, optional-->
+                <funding_acknowledgement_text>$Funding_Acknowledgement['funding_acknowledgement_text']$</funding_acknowledgement_text> <!--user input, optional-->
+            </Funding_Acknowledgement>
+        $END_IF
+        </Citation_Information>
+    $END_IF
+    $IF(Modification_Detail)
+        <Modification_History> <!--user input, recommended-->
+            $FOR(field, k=Modification_Detail)
+            <Modification_Detail>
+                <modification_date>$field['modification_date']$</modification_date> <!--user input, recommended-->
+                <version_id>$field['version_id']$</version_id>   <!--user input, recommended-->
+                <description>$field['description']$</description> <!--user input, recommended-->
+            </Modification_Detail>
+            $END_FOR
+        </Modification_History>
+     $END_IF
+        <License_Information>
+            <name>Creative Common Public License CC0 1.0 (2024)</name>
```

Comment:
Are all PDS4 labels licensed this way? Would it ever make sense for a user to produce a proprietary label?

Reply:
This was done on the advice of Mitch. I imagine if a proprietary label did need to be made, they could add in a `License_Information` section on their own. Currently the template does not create `License_Information` sections.

Reply:
With rare exceptions, I expect CC0 to be the license specified in labels.  However, we should always allow for other values, because they may occur.  For example, some data from the CLPS program come from commercial providers and might not be CC0, though RMS is unlikely to deal with such data.